### PR TITLE
More explicit error message when missing .d.ts files

### PR DESCRIPTION
**Pull request by shmidt-i**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by shmidt-i** [comment]
Bumping the package due to the https://github.com/egoist/rollup-plugin-esbuild/issues/27 , was failing the CI build

File: package.json [hunk 1]
```diff
@@ -53,5 +53,8 @@
     "*.{json,md}": [
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "rollup-plugin-esbuild": "2.0.0"
```
Comment (freben) [inline package.json:58]: this should be on cli though, right?

Comment (Rugvip) [reply package.json:58]: Yep

Comment (shmidt-i) [reply package.json:58]: Yep, was pretty sure I ran it in the cli package, gonna move it there

**Review by Rugvip — APPROVED** [review]

**Pull request merged**